<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PVContourWidget</class>
 <widget class="QWidget" name="PVContourWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>298</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,0" columnstretch="1,0">
   <item row="0" column="0">
    <widget class="QListView" name="isoValuesListView"/>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0,0">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>First Value</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="firstValueLineEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Last Value</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lastValueLineEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Step</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="stepValueEdit"/>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="applyButton">
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
